<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptWindow</class>
 <widget class="QMainWindow" name="CryptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_to"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pb_do">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_to">
      <property name="text">
       <string>Receiver Public Key</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="l_my">
      <property name="text">
       <string>My Private Key</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="cb_my"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="pte_src"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="pte_dst"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pb_do</sender>
   <signal>clicked()</signal>
   <receiver>CryptWindow</receiver>
   <slot>encrypt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>encrypt()</slot>
 </slots>
</ui>
